app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def calculate_team_total_rating(player: list) -> int:
+    return sum([player.get_rating() for player in player])
```

Comment:
The list comprehension `[player.get_rating() for player in player]` is unnecessary here. You can pass a generator expression directly to `sum`, like this: `sum(player.get_rating() for player in player)`. This will prevent creating a redundant list and improve code efficiency as per the checklist guidelines .